src/System.Private.Uri/src/System/UriFormatException.cs

```diff
@@ -2,12 +2,22 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if netstandard10
```

Comment:
OK perhaps netstandard1.0 is messing me up then. How does this compile for that version but not a netstandard1.3 version?